CRM/Chreports/Upgrader.php

```diff
@@ -1121,4 +1121,49 @@ public function upgrade_102009() {
     return TRUE;
   }
 
+  public function upgrade_102010() {
+    $this->ctx->log->info('Reporting v1.2.3 (#010): Add "Batch Name" Filter to Report Filter list & Template Columns');
+    $non_migrated_templates = E::getNonMigratedReportTemplates();
+    // Fetch all saved reports
+    $reportInstances = civicrm_api3('ReportInstance', 'get', [
+      'sequential' => 1,
+      'created_id' => ['IS NOT NULL' => 1],
+    ]);
+    if($reportInstances) {
+      // Iterate through all the reports
+      foreach($reportInstances['values'] as $report) {
+        if(!empty($report['created_id'])) {
+          if(!in_array($report['report_id'], $non_migrated_templates)) {
+            $filePath = CRM_Chreports_Reports_ReportConfiguration ::getFilePath($report);
+            if (is_file($filePath['source'])) {
+              $status = false;
```

Comment:
`status` is a vague term that provides no context, and makes it more difficult to read code. To understand what it meant and what it was for, I had to look at the rest of the code, whereas if it was named `updateReport` or similar, it becomes clear right away.

```suggestion
              $updateReport = false;
```